packages/react-native-web/src/exports/ScrollView/ScrollViewBase.js

```diff
@@ -122,17 +124,19 @@ export default class ScrollViewBase extends Component<*> {
       ...other
     } = this.props;
 
+    const showScrollIndicator = showsHorizontalScrollIndicator && showsVerticalScrollIndicator;
     return (
       <View
         {...other}
         onScroll={this._handleScroll}
         onTouchMove={this._createPreventableScrollHandler(this.props.onTouchMove)}
         onWheel={this._createPreventableScrollHandler(this.props.onWheel)}
         ref={this._setViewRef}
-        style={StyleSheet.compose(
+        style={StyleSheet.flatten([
```

Comment:
`flatten` is a deopt.

```suggestion
style={[
  style,
  !scrollEnabled && styles.scrollDisabled,
  !showScrollIndicator && style.disableScrollIndicator
]}
```